src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in

```diff
@@ -60,7 +61,8 @@ def predict(schema_madlib, model_table, test_table, id_col, model_arch_table,
     input_shape = get_input_shape(model_arch)
     compile_params = "$madlib$" + compile_params + "$madlib$"
     model_summary_table = add_postfix(model_table, "_summary")
-    class_values, _ = get_class_values_and_type(model_summary_table)
+    class_values, _ = get_col_value_and_type(
```

Comment:
Instead of calling `get_col_value_and_type()` here, can we do something similar to what we have in [madlib_keras.py_in:202](https://github.com/apache/madlib/pull/365/files#diff-4d3eba8c7b20343818ac24e5730779e7R202)